docs/install/knative-with-operators.md

```diff
@@ -25,10 +25,58 @@ Disk storage to at least 30 GB. If you have multiple nodes for your cluster, set
 
 ## Limitations of Knative Operators:
 
-Knative Operators use custom resources (CRs) to configure your Knative deployment.
+Knative Operators are still in Alpha phase. They have not been tested in a production environment, and should be used
+for development or test purposes only.
 
- - Currently, the CRs included with Knative Operators do not provide high availability (HA) capabilities.
- - Knative Operators have not been tested in a production environment, and should be used for development or test purposes only.
+## Option 1: Install Knative with combined operator
+
+Knative has released a combined operator to install both Serving and Eventing components. You can find the information
+about the releases on the [Releases page](https://github.com/knative-sandbox/operator/releases).
+
+### Installing the combined Knative Operator
+
+__From releases__:
+
+Run the following command to install the combined Knative Operator:
+
+```
+kubectl apply -f https://github.com/knative-sandbox/operator/releases/download/v0.14.1/operator.yaml
```

Comment:
Do we have some "latest" link now instead of calling out 0.14.1 specifically?
cc @jcrossley3